tokens/properties/alias.json

```diff
@@ -56,6 +56,9 @@
                 },
                 "description": {
                     "value": "{font.weight.regular.value}"
+                },
+                "link": {
+                    "value": "{font.weight.regular.value}"
```

Comment:
Possible solution: have a "normal" and a "strong" version of links with regular and bold font-weights respectively